src/util/swapHelpers.ts

```diff
@@ -160,6 +160,10 @@ export async function makeSwapPluginQuote(
           savedAction.orderUri = `${savedAction.orderUri}${tx.txid}`
       }
 
+      if (quoteId == null && (swapInfo.isDex ?? false)) {
```

Comment:
Doing `(swapInfo.isDex === true)` also avoids the lint error, but makes more sense when you read it.

Reply:
agreed, will fix